src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/StegService.kt

```diff
@@ -274,7 +283,7 @@ class StegService(
                 return BehandlerRolle.SAKSBEHANDLER
             }
         }
-        return saksbehandling.steg.tillattFor
+        return stegType.tillattFor
```

Comment:
Tidligere validerte vi steget behandlingen stod i, men blir det ikke mer riktig å validere steget som skal utføres?

Reply:
stegType er "neste" steg og saksbehandling.steg er "gjeldende steg"? 

Isåfall:
Saksbehandling.steg => send_til_beslutter
Stegtype => Beslutte_vedtak

Da bør vi jo validere at beslutte_vedtak er tillatt for saksbehandleren... 

Jeg sier "ja", men vil gjerne ha en test som feiler/fikser dette 😬